Python/instrumentation.c

```diff
@@ -1216,7 +1208,9 @@ _Py_call_instrumentation_line(PyThreadState *tstate, _PyInterpreterFrame* frame,
         }
     } while (tools);
     Py_DECREF(line_obj);
+    uint8_t original_opcode;
```

Comment:
why not inline the declaration?

Reply:
Because C doesn't allow a declaration immediately following a label.
For arcane historical reasons, probably.